### DISCUSSION TITLE
AUR - v0.5.0 update

### DISCUSSION THREAD

**Discussion by Gusted** [emersion/maddy#383] [Q&A] [answered]
As @foxcpp manages the AUR package, I think it's better to comment here.

Could the AUR be updated to v0.5.0?
A simple bump in the pkgver and update of the shashum doesn't work because the source of v0.5.0 wasn't yet uploaded to https://maddy.email/builds/0.5.0/

**Comment by foxcpp** [comment, marked as answer]
Done. Thanks for reminding me.
